app/components/animated_number/index.tsx

```diff
@@ -36,14 +36,16 @@ const AnimatedNumber = ({
     const prevNumberersArr = Array.from(prevNumberString, Number);
 
     const [numberHeight, setNumberHeight] = React.useState(0);
-    const animations = useMemo(() => numberStringToDigitsArray.map((__, index) => {
-        if (typeof prevNumberersArr[index] !== 'number') {
-            return new Animated.Value(0);
-        }
+    const animations = React.useRef(
```

Comment:
Right now, on every render, we are executing the map, which is not great. The React recommendation is to add an `if null` to initialize the value:
https://react.dev/reference/react/useRef#avoiding-recreating-the-ref-contents

Not a big fan, but if it is the recommended way from react, let's use it.